File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {

       	int n = nums.length;
		
		
		HashSet<Integer> set1= new HashSet<>();
		for(int i =0;i<n;i++) {
			int num = nums[i];
			if(set1.contains(num)) {
				return true;
			}
			set1.add(num);
		}

		return false;

        
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {

    let n = nums.length;
    const seen = new Set();
    for(let i =0;i<n;i++){
        if(seen.has(nums[i])){
            return true;
        }
        seen.add(nums[i]);
    }
    return false;
    
};